database/migrations/2018_03_15_124536_add_description_to_nodes.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddDescriptionToNodes extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('nodes', function (Blueprint $table) {
+            $table->text('description')->nullable()->after('name');
```

Comment:
You cannot have a `TEXT` field that is nullable in MySQL.

Reply:
Oh, didn't know about that. Thanks.